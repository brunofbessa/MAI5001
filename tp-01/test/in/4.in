I
123
produto1
1.83
I
456
produto2
0.10
I
321
produto3
0.33
R
456
R 
111
L
F
