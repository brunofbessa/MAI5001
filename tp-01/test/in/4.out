Ok
Ok
Ok
Ok
Erro
123 - produto1 - 1.83
321 - produto3 - 0.33
Fim
